docs/05-staking/01-what_is_staking.md

```diff
@@ -0,0 +1,9 @@
+---
+title: What is staking
+---
+
+## What is staking?
+Staking is the mechanism a Proof of Stake (PoS) blockchain offers to participate actively in the network consensus.
+To do so, a user has to lock a certain amount of coins; by doing so they become _Forger_. The higher the amount staked, the greater the chances of being able to create (_forge_) a block. 
+As long as the amount is staked, the user cannot dispose of it. \
```

Comment:
The `\` is rendering is as is. (See https://dd4f0441.evm-documentation.pages.dev/staking/what_is_staking)
You can replace it with [two spaces](https://itsfoss.com/markdown-new-line/#:~:text=To%20add%20a%20new%20line%20in%20Markdown%2C%20the%20proper%20Markdown,press%20the%20enter%20key%20twice.), or use `<br/>` instead